run_dir/static/js/sample_requirements.js

```diff
@@ -0,0 +1,1216 @@
+                  <p>
+                    <span class="fw-bold">Last modified</span><span> by {{this.$root.draft_sample_requirements["Last modified by user"]}} at </span><span class="fst-italic"> {{draft_last_modified_at}}</span>
+                    ---
+                    <span class="fw-bold">Version {{published_version}} was published </span><span> by {{published_issued_by}} at </span><span class="fst-italic"> {{published_at}}</span>
+                  </p>
+                  <template v-if="draft_locked_by_someone_else">
+                    <a class="btn btn-danger" @click="reassign_lock"><i class="fas fa-user-lock"></i> Reassign lock to you</a>
+                    <p v-if="draft_locked_by_someone_else">Draft is currently locked by {{draft_locked_by}}</p>
+                  </template>
+                </div>
+              </template>
+              <template v-else>
+                <h1>
+                  <span id="page_title">New Sample Requirements</span>
+                </h1>
+                <p>
+                <span class="fw-bold">Version {{published_version}} was published </span><span> by {{published_issued_by}} at </span><span class="fst-italic"> {{published_at}}</span>
+                </p>
+                <p> No draft sample requirements exists. </p>
+                <p><button class="btn btn-success" @click="create_new_draft"><i class="fas fa-user-lock"></i> Create New Draft</button></p>
+              </template>
+            </div>
+            <template v-if="draft_exists">
+              <div class="row">
+                <div class="col-12">
+                  <v-draft-changes-list :modal="false"/>
+                </div>
+              </div>
+            </template>
+
+            <template v-if="draft_exists">
+              <div class="mt-2">
+                <div class="row" id="table_h_and_search">
+                  <h2 class="col mr-auto">Preview of Draft</h2>
+                </div>
+                <v-requirements-table/>
+              </div>
+            </template>
+
+            <template v-if="draft_exists">
+              <!--- Modals --->
+              <div class="modal fade" id="publish_draft_modal" tabindex="-1" role="dialog" aria-labelledby="publishDraftModalHeader">
+                <div class="modal-dialog" role="document">
+                  <div class="modal-content">
+                    <div class="modal-header">
+                      <h4 class="modal-title" id="publishDraftModalHeader">Publish new sample requirements</h4>
+                      <button type="button" class="btn-close" data-dismiss="modal" aria-label="Close"></button>
+                    </div>
+                    <div class="modal-body">
+                      <template v-if="this.$root.no_validation_messages">
+                        <p>Are you sure you want to publish the current draft?</p>
+                        <p>This will then become the default cost calculator used for all quotes.</p>
```

Comment:
Should it be sample requirements rather  than cost calculator?

Reply:
Yes! Thank you! Fixed it now.